runtime/browser/linux/xwalk_notification_manager.cc

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/browser/linux/xwalk_notification_manager.h"
+
+#include <map>
+
+#include "base/strings/utf_string_conversions.h"
+#include "content/public/browser/browser_thread.h"
+#include "content/public/browser/desktop_notification_delegate.h"
+#include "content/public/browser/web_contents.h"
```

Comment:
"utf_string_conversions.h" is for "base::UTF16ToUTF8"
"browser_thread.h" is for "content:BrowserThread::PostTask"
"desktop_notification_delegate.h" is for "content::DesktopNotificationDelegate"
"platform_notification_data.h" is for "content::PlatformNotificationData"